src/borg_import/main.py

```diff
@@ -37,11 +37,19 @@ def borg_import(args, archive_name, path, timestamp=None):
 
 def list_borg_archives(args):
     borg_cmdline = ['borg', 'list', '--short']
-    repository = args.repository.resolve()
+    repository = repo_resolve(args.repository)
     borg_cmdline.append(str(repository))
     return subprocess.check_output(borg_cmdline).decode().splitlines()
 
 
+def repo_resolve(repository):
+    if '@' in str(repository):
+        # looks like a remote repo - don't try and resolve the path
+        return repository
+    else:
+        return repository.resolve()
```

Comment:
https://docs.python.org/3/library/pathlib.html#pathlib.Path.resolve

hmm, maybe we could check whether this is needed at all.

Reply:
check whether we change the current working directory (cwd) somewhere in the code...

Reply:
@ThomasWaldmann, why would you need to check for cwd changes?

Reply:
I see no cwd or chdir in the code, but maybe you want to take a closer look at this before we get rid of `repository.resolve()`.